examples/jsm/loaders/MMDLoader.js

```diff
@@ -1519,6 +1533,23 @@ class MaterialBuilder {
 
 			}
 
+			if ( texture instanceof CompressedTexture ) {
```

Comment:
It'd be better to do this check instead:

```js
if ( texture.isCompressedTexture === true ) {
```

Reply:
Hi, @mrdoob 
should I just push on this branch or make another PR?

Reply:
You can just push to this branch 👍

Reply:
updated, plz take a look, thx!